firefox-ios/Client/Frontend/Browser/WebView/WebviewViewController.swift

```diff
@@ -10,7 +10,7 @@ class WebviewViewController: UIViewController, ContentContainable, Screenshotabl
     private var webView: WKWebView
     var contentType: ContentType = .webview
 
-    init(webView: WKWebView, isPrivate: Bool = false) {
```

Comment:
I stumbled on this unused `private` parameter, so taking this opportunity to remove it